src/test/java/htsjdk/samtools/SamReaderFactoryTest.java

```diff
@@ -538,55 +536,48 @@ public void testSamReaderFromMalformedSeekableStream() throws IOException {
 
     @Test(singleThreaded = true, groups="unix")
     public void testWriteAndReadFromPipe() throws IOException, InterruptedException, ExecutionException, TimeoutException {
+        final SAMRecordSetBuilder builder = new SAMRecordSetBuilder(false, SortOrder.unsorted);
+        for (int i=1; i<= 5000; ++i) {
```

Comment:
format code to have spaces around operators, e.g. `int i = 1; i < = 5000`

or use `IntStream.range(1, 5000).forEach(i -> builder.addPair("q" + i, 0, i, i));`